functions-and-scope.js

```diff
@@ -16,6 +17,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+//Stappenplan:
+//Stap 1 loopen door array
+//Stap 2 length toevoegen aan loop zodat hij werkt ongeacht de hoeveelheid cijfers
+//stap 3 Aantal cijfer 8 of hoger selecteren
+//Stap 4 Cijfers opslaan
+//stap 4 cijfers loggen
+
+const cumLaudeList = [];
+
+for (let i = 0; i < grades.length; i++) {
+    if (grades[i] >= 8)  {
+        cumLaudeList.push(grades[i]);
```

Comment:
Met de .push je een nieuw item naar een array, en daarbij mag je geen bestaande methode hiervoor gebruiken! (Stond in de opdracht ;)). Maar wel een leuke oplossing.
Het resultaat hadden we ook simpelweg kunnen behalen door het nieuwe cijfer op te slaan als hoogste waarde:

`let cumLaudeList = 0;
     for (let i=0; i < grades.length; i++){
            if(grades[i]) >= 8)
             cumLaudeList++ 
}
}`